scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -319,6 +319,11 @@ class FossId(
             val filteredPackages = packages
                 .filter { packageNamespaceFilter.isEmpty() || it.id.namespace == packageNamespaceFilter }
                 .filter { packageAuthorsFilter.isEmpty() || packageAuthorsFilter in it.authors }
+                .onEach {
+                    if (it.vcsProcessed.path.isNotEmpty()) {
+                        log.warn { "Ignoring package with url ${it.vcs.url} with non-null path ${it.vcs.path}" }
+                    }
+                }.filter { it.vcsProcessed.path.isEmpty() }
```

Comment:
According to my experience, generally the whole repository is scanned with another project-package, therefore we can safely ignore these packages based on a sub-path of the repository.
However putting an `OrtIssue` is a good idea to track such filtering. But I don't want some pipeline to fail because such skips were done. Therefore I suggest we put an `OrtIssue` but with severity set to `HINT`.
Would it be acceptable ?

Reply:
Yes, this would be fine with me.